pkg/configmap/wsvalidator/wsvalidator.go

```diff
@@ -0,0 +1,23 @@
+package wsvalidator
```

Comment:
_A parsing failure is also a validation error._

Consider `wsparser` instead of `wsvalidator` to follow the existing `wiparser` naming convention.

Reply:
winInstParser & winSerParser ??